spec.html

```diff
@@ -89,133 +92,133 @@ <h3>Non-terminals</h3>
                   <tr class="section">
                     <td class="fix nt">element</td>
                     <td class="fix op">::=</td>
-                    <td class="fix ex">"\x01" e_name double</td>
+                    <td class="fix ex">signed_byte(1) e_name double</td>
                     <td>64-bit binary floating point</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x02" e_name string</td>
+                    <td class="fix ex">2 e_name string</td>
                     <td>UTF-8 string</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x03"&nbsp;e_name&nbsp;document</td>
-                    <td>Embedded document</td>
+                    <td class="fix ex">signed_byte(3) e_name document</td>
+                    <td>Embedded document or object</td>
```

Comment:
This page uses the terminology "document" consistently instead of "object". I would revert this edit.